packages/reporting/tests/unit/reporting_templates.test.mjs

```diff
@@ -62,6 +62,10 @@ describe("build reporting templates", () => {
 
 		it("should not include GitHub workflow run link in the report when not running in GitHub Actions", () => {
 			// GIVEN
+			vi.stubEnv("GITHUB_REPOSITORY", "");
```

Comment:
I added these because these variables are set when running in the CI, otherwise [it would fail](https://github.com/aws-powertools/actions/actions/runs/8969113794).

Reply:
thank you, totally missed that when running in Actions.

We only need `GITHUB_ACTIONS` to not be set, everything else should have a sane default. Removing so they can be tested too.